docs/music-providers/deezer.md

```diff
@@ -1,21 +1,20 @@
 # Deezer Provider ![Preview image](../assets/icons/deezer-icon.svg){ width=70 align=right }
 
-Music Assistant has full support for [Deezer](https://www.deezer.com/) as Music Provider added in version 2.0.0 beta 30, contributed and maintained by @micha91 and @arctixdev.
+Music Assistant has support for [Deezer](https://www.deezer.com/) as a Music Provider. This was added in version 2.0.0 beta 30. Contributed and maintained by @micha91 and @arctixdev.
 
-!!! tip
+!!! tip "Note"
     - Because of Deezer's TOS we only support HiFi/Premium/Family accounts.
     - It is normal that syncing all your items from Deezer takes some time.
 
 ## Supported features
-- Support for "library" Artist, Albums, Tracks and Playlists.
-- Items in your Deezer library will be marked as "Favorites" too in Music Assistant.
-- Marking an item as "Favorite" from the Music Assistant interface will also mark it as favorite in Deezer.
-- Playback of Deezer content to all players supported by Music Assistant in FLAC HiFi.
 - Searching from the Music Assistant interface support Artists, Albums, Tracks and Playlists.
-- Full Artist, Album, Track and Playlist metadata.
-- Playlist creation + adding and removing tracks from playlists.
+- Items in your Deezer library will be marked as a "Favorite" in Music Assistant.
+- Marking an item as a "Favorite" from the Music Assistant interface will also mark it as a favorite in Deezer.
+- Playback of Deezer content to all players supported by Music Assistant is in FLAC HiFi.
```

Comment:
Actually i dont think its hifi, I think i midunderstood Micha when i originally wrote the discussion. Otherwise it looks very good

Reply:
Ok we can probably delete that point then as that is standard across all providers?

Reply:
@arctixdev also what do you think about the use of the word favourite? Is that correct or is it actually items that are added to the MA library are added to the Deezer Library (do they call it that?) and vica versa?

Reply:
Sorry i didnt repond, i was very busy with some pre-exams in school. Yes, favorite is correct. Very cool you improved the text :smiley: I can also update the discussion if you didnt already do so

Reply:
Good luck in the exams! Don’t worry about the discussion as we will be removing all of that when this goes live. 
So to confirm my understanding. Things in Deezer are “favourites”. Any favourites in Deezer are added to the MA Library and then marked as a favourite in MA. If you add something from Deezer to the MA library then nothing will happen in Deezer unless you also mark it as a favourite (at which time the thing will be added to the Deezer favourites). Is that right?

Reply:
Yes that is correct :+1: Thanks!